cpu/kinetis/periph/uart.c

```diff
@@ -296,7 +305,19 @@ static inline void uart_init_lpuart(uart_t uart, uint32_t baudrate)
     LPUART_Type *dev = uart_config[uart].dev;
     uint32_t clk = uart_config[uart].freq;
 
-    /* Remember to select a module clock in board_init! (SIM->SOPT2[LPUART0SRC]) */
+    /* Set LPUART clock source */
+#ifdef SIM_SOPT2_LPUART0SRC
+    if (dev == LPUART0) {
+        SIM->SOPT2 = (SIM->SOPT2 & ~SIM_SOPT2_LPUART0SRC_MASK) |
+            SIM_SOPT2_LPUART0SRC(LPUART_0_SRC);
+    }
+#endif
+#ifdef SIM_SOPT2_LPUART1SRC
```

Comment:
Though I see a define for `SIM_SOPT2_LPUART0SRC`, I can't find for `SIM_SOPT2_LPUART1SRC`. Is this intentional? Same for `SIM_SOPT2_LPUART1SRC_MASK` used below, since there is a definition for `SIM_SOPT2_LPUART0SRC_MASK`.

Reply:
LPUART1SRC only exist for CPUs which have two or more lpuart modules (LPUART0, LPUART1). Therefore we need the preprocessor conditional to avoid compilation errors on the CPUs which don't have these modules.

Reply:
Ok I see, that's right for me.